### DISCUSSION TITLE
Updating assignments after creation?

### DISCUSSION THREAD

**Discussion by nrmn4usm** [community/community#38297] [GitHub Education]
Hi,
Is there any good solution to the thread pointed by https://education.github.community/t/updating-assignments-after-creation/23840 from GitHub Education Community that is now read-only? The question were asked in 2018, last responded by acrokat (Katherine Kampf) in Sept. 2021 saying that the solution will be provided in 2021, and now already Nov. 2022.
Or maybe I missed the solution from somewhere? All the solutions provided there were too difficult to understand (some contains file that no longer there) and I was happy to see that acrokat (Katherine Kampf), a GitHub Staff understand the problem and promised to provide a solution.

**Comment by ossanna16** [comment]
Hi there @nrmn4usm and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
